cargo-dist/src/config/v1/mod.rs

```diff
@@ -481,6 +483,173 @@ impl TomlLayer {
             }
         }
     }
+
+    /// Determines whether the configured install paths are compatible with each other
+    pub fn validate_install_paths(&self) -> DistResult<()> {
+        if let Some(installers) = &self.installers {
+            if let Some(paths) = &installers.common.install_path {
+                if paths.len() > 1 && paths.contains(&InstallPathStrategy::CargoHome) {
+                    return Err(DistError::IncompatibleInstallPathConfiguration {});
+                }
+            }
+        }
+
+        Ok(())
+    }
+
+    // We say "value is being ignored", but it seems like we use the "ignored" values anyway?
```

Comment:
can you give more details on how it is used? that def seems bad!! but agree we should repro for now, might leave a TODO in this comment to return to this